include/srf/node/rx_sink.hpp

```diff
@@ -49,7 +50,10 @@ template <typename T, typename ContextT>
 class RxSink : public RxSinkBase<T>, public RxRunnable<ContextT>, public RxPrologueTap<T>
 {
   public:
-    using observer_t = rxcpp::observer<T>;
+    using observer_t       = rxcpp::observer<T>;
+    using on_next_fn_t     = std::function<void(T)>;
+    using on_error_fn_t    = std::function<void(std::exception_ptr)>;
+    using on_complete_fn_t = std::function<void()>;
```

Comment:
these don't appear to be used immediately `RxSink`, would they be better in `RxSinkBase`?

Reply:
I guess `RxSink` is probably the right choice, since the `RxSource` observer is used in `RxNode`